src/org/ods/usecase/LeVADocumentUseCase.groovy

```diff
@@ -1105,8 +1105,15 @@ class LeVADocumentUseCase extends DocGenUseCase {
 
         def documentType = DocumentType.DTR as String
 
-        def uri = this.createOverallDocument("Overall-Cover", documentType, metadata, null, this.getWatermarkText(documentType))
-        this.updateJiraDocumentationTrackingIssue(documentType, "A new ${documentTypeName} has been generated and is available at: ${uri}.")
+        // Same behaviour to get sections not done as DTR
+        def sections = this.jiraUseCase.getDocumentChapterData(documentType)
```

Comment:
@hrcornejo @martsec why does an overall document require document chapters? The template is only a cover page and then we attach the concrete repo-level docs. Thoughts?

Reply:
We need the chapters to know the status and put or not the WIP watermark in Overall documents. See PLTF-924

Reply:
The overall DTR has no chapters. To achieve what you want to achieve, you could set a flag on repo.data.isXYZ or hasXYZ and then query the state here. You don't need to fetch all the chapter data another time.

Reply:
Added all sectionsNotDone. We need the info to generate a right message:
`repo.data.sectionsNotDoneDTR = `sectionsNotDone``

Reply:
@metmajer The thing here is that we need to apply the WIP watermark if there are chapters of the DTRs that are not in done. We have detected that the individual DTRs had the WIP watermark but this was not applied in the overall DTR. Since this last document is the one uploaded (and thus, more valid), we need a way to know when to add the watermark

Reply:
Sorry @metmajer I don't understand where to add that flag. We need the list of pending sections to put it message.

Reply:
Finally stored in: this.project.data.sectionsNotDoneXYZ